### PR TITLE
Adding vuid 00361

### PR DESCRIPTION
**Pull request by ziga-lunarg**
Check if the command pool the command buffer was allocated from supports the used pipelineBindPoint

### CODE REVIEW

File: layers/core_validation.cpp [hunk 1]
```diff
@@ -7101,6 +7101,13 @@ bool CoreChecks::PreCallValidateCmdBindDescriptorSets(VkCommandBuffer commandBuf
                          "(%zu) when pipeline layout was created",
                          firstSet, setCount, pipeline_layout->set_layouts.size());
     }
+
+    static const std::map<VkPipelineBindPoint, std::string> bindpoint_errors = {
+        std::make_pair(VK_PIPELINE_BIND_POINT_GRAPHICS, "VUID-vkCmdBindDescriptorSets-pipelineBindPoint-00361"),
+        std::make_pair(VK_PIPELINE_BIND_POINT_COMPUTE, "VUID-vkCmdBindDescriptorSets-pipelineBindPoint-00361"),
+        std::make_pair(VK_PIPELINE_BIND_POINT_RAY_TRACING_NV, "VUID-vkCmdBindDescriptorSets-pipelineBindPoint-00361")};
```
Comment (sfricke-samsung) [inline layers/core_validation.cpp:7108]: nit, can we use `VK_PIPELINE_BIND_POINT_RAY_TRACING_KHR` here (granted it isn't else where, maybe worth fixing here or in separate PR)

Comment (ncesario-lunarg) [reply layers/core_validation.cpp:7108]: +1

We currently have both NV and KHR used here, but regardless I'm in favor of using the KHR one going forward (it looks like most of the other NV usage in this file should change to KHR, but that would be for a separate PR).

**Review by ncesario-lunarg — APPROVED** [review]
Just the one nit, but LGTM!

File: layers/core_validation.cpp (hunk 1 above)
Comment (sfricke-samsung) [reply layers/core_validation.cpp:7108]: agree, leave for now, worry bout it in seperate PR

**Pull request merged**